test/scripts/lint.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Sorry. Surprised Git didn't pickup this file move. I guess I made too many changes at the same time as the move, so it lost track 🤷🏼‍♂️